modules/apps/product-navigation/product-navigation-control-menu-theme-contributor/src/main/resources/META-INF/resources/control_menu/_control_menu.scss

```diff
@@ -76,6 +76,7 @@ html#{$cadmin-selector} {
 		.control-menu-level-1-heading {
 			font-size: 16px;
 			font-weight: 600;
+			margin: 0px;
```

Comment:
@rebsilva the SF error is likely because of this. For 0 css values we don't need units. Also, it's better to only override the style needed, which I believe would only be margin-bottom instead of all margins. Can you update this and retest with CI?

Reply:
Sure! I will do this now, thanks!